pkgs/development/python-modules/dynalite-devices/default.nix

```
@@ -37,6 +37,9 @@ buildPythonPackage rec {
     "dynalite_devices_lib"
   ];
 
+  # it would use the erroneous tag v0.47
+  passthru.skipBulkUpdate = false;
```

Oops.
```suggestion
passthru.skipBulkUpdate = true;
```
fixed by https://github.com/NixOS/nixpkgs/pull/281444